libs/auth-api-lib/src/lib/delegations/delegations-incoming.service.ts

```diff
@@ -72,7 +84,11 @@ export class DelegationsIncomingService {
     }
 
     // Index incoming delegations
-    void this.delegationsIndexService.indexDelegations(user)
+    try {
+      void this.delegationsIndexService.indexDelegations(user)
+    } catch {
+      this.logger.error('Failed to index delegations')
+    }
```

Comment:
This try/catch in unnecessary as the `indexDelegations` function is `async` and the `Promise` is not awaited.

But something like 

```
void this.delegationsIndexService
        .indexDelegations(user)
        .catch((error) => 
             this.logger.error('Failed to index delegations', error)
         )
```

should log it if it throws.

Reply:
Fixed.